crates/bevy_winit/src/lib.rs

```diff
@@ -276,8 +136,31 @@ where
     panic!("Run return is not supported on this platform!")
 }
 
-pub fn winit_runner(app: App) {
-    winit_runner_with(app);
+#[derive(SystemParam)]
+struct WindowEvents<'w, 's> {
+    window_resized: EventWriter<'w, 's, WindowResized>,
+    window_close_requested: EventWriter<'w, 's, WindowCloseRequested>,
+    window_scale_factor_changed: EventWriter<'w, 's, WindowScaleFactorChanged>,
+    window_backend_scale_factor_changed: EventWriter<'w, 's, WindowBackendScaleFactorChanged>,
+    window_focused: EventWriter<'w, 's, WindowFocused>,
+    window_moved: EventWriter<'w, 's, WindowMoved>,
+}
+
+#[derive(SystemParam)]
+struct InputEvents<'w, 's> {
+    keyboard_input: EventWriter<'w, 's, KeyboardInput>,
+    character_input: EventWriter<'w, 's, ReceivedCharacter>,
+    mouse_button_input: EventWriter<'w, 's, MouseButtonInput>,
+    mouse_wheel_input: EventWriter<'w, 's, MouseWheel>,
+    touch_input: EventWriter<'w, 's, TouchInput>,
+    // mouse_motion: EventWriter<'w, 's, MouseMotion>,
```

Comment:
```suggestion
```
Remove this if not being used?

Reply:
going to re-add this, not entirely sure why it was removed